general-concepts/manifest/text.xml

```diff
@@ -39,6 +39,42 @@ action from developers.
 </p>
 </body>
 </subsection>
+
+<subsection>
+<title>Updating Manifests</title>
+<body>
+<p>
+Updating existing entries within a Manifest must be done with care. Upstream
```

Comment:
I think that throughout this should either be spelled `manifest` in lower case if you refer to the concept, or `the <c>Manifest</c> file` if you refer to the actual file.

```suggestion
Updating existing entries within a <c>Manifest</c> file must be done with care. Upstream
```